reconcile/prometheus_rules_tester.py

```diff
@@ -275,6 +306,8 @@ def check_prometheus_tests(tests, rules, thread_pool_size, settings):
     for path, test in tests.items():
         test_to_run = {"path": path}
 
+        target_clusters = get_target_clusters_from_jinja_test_template(test)
+
```

Comment:
Since clusters are simple strings, we should check they actually correspond to real clusters and fail if the cluster does not exist. I don't think we can do that while bundling because refs are not resolved in resources, so we will need to do it here.